torchao/prototype/moe_quant/quantizable_moe_modules.py

```diff
@@ -162,13 +224,28 @@ def group_tokens_by_expert(
                 w1 = self.w1[expert]  # I, D
                 w2 = self.w2[expert]  # D, I
                 w3 = self.w3[expert]  # I, D
+                if self.gpt_oss_mlp:
```

Comment:
this seems pretty hacky, I'd suggest to try to extend based on our long term API instead